src/btree/bt_io.c

```diff
@@ -146,23 +138,23 @@ __wt_bt_read(WT_SESSION_IMPL *session,
 	WT_STAT_CONN_INCRV(session, cache_bytes_read, dsk->mem_size);
 	WT_STAT_DATA_INCRV(session, cache_bytes_read, dsk->mem_size);
 	WT_STAT_SESSION_INCRV(session, bytes_read, dsk->mem_size);
-	(void)__wt_atomic_add64(
-	    &S2C(session)->cache->bytes_read, dsk->mem_size);
+	(void)__wt_atomic_add64(&S2C(session)->cache->bytes_read, dsk->mem_size);
 
 	if (0) {
-corrupt:	if (ret == 0)
+	corrupt:
```

Comment:
`goto` labels get indented which isn't great. However it should be trivial to write a script that flushes these labels to the left.

Reply:
Lint prefers them on separate lines, too.